docs/openapi.yml

```diff
@@ -2101,6 +2098,16 @@ components:
         summary:
           $ref: "#/components/schemas/marketSummary"
           description: Financial summary for the market
+    marketTokenBreakdown:
+      type: object
+      description: Details of tokens that are tied to a market
+      properties:
+        crs:
+          $ref: "#/components/schemas/tokenResponse"
+          description: Details of the CRS token
+        staking:
+          $ref: "#/components/schemas/tokenResponse"
```

Comment:
this one would be a marketTokenResponse

Reply:
I'm using `MarketTokenDto` so that we get the market token summary, then mapping to `TokenResponseModel` so that the `market` and `liquidityPool` address are not repeated in the response